src/components/Tasks.tsx

```diff
@@ -1,18 +1,180 @@
-import React from 'react';
+import React, { useEffect, useState, useRef } from 'react';
+import deleteImage from '../assets/img/delete.png';
+import './Tasks.css'
+
+const tasksLocalStorageKey = "studysync-tasks"
+
+type TaskData = {
+  taskString: string;
+  taskCompleted: boolean;
+}
+
+// TODO: https://stackoverflow.com/questions/5364062/how-can-i-save-information-locally-in-my-chrome-extension
+// Chrome storage: https://developer.chrome.com/docs/extensions/reference/api/storage
+// Firefox storage: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/storage
+
+// get the task string and check from the new task thingie
+function getNewTask(): [string, boolean] {
+  const taskString = document.getElementById(
+    'new-task-string'
+  ) as HTMLInputElement;
+  let tString = taskString ? taskString.value : '';
```

Comment:
More accurate to declare variables only used once with `const`
```suggestion
  const tString = taskString ? taskString.value : '';
```